soc/arm/st_stm32/stm32f0/Kconfig.defconfig.stm32f030x4

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Antony Pavlov <antonynpavlov@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F030X4
+
+# N.B. stm32cube/stm32f0xx/soc/stm32f0xx.h know nothing
```

Comment:
I had a look to cube and it indicates the following:
`  /* #define STM32F030x6 */  /*!< STM32F030x4, STM32F030x6 Devices (STM32F030xx microcontrollers where the Flash memory ranges between 16 and 32 Kbytes)              */
`
So you're doing exactly what is indicated by Cube. To minimize confusion, can you change your comment to something like:
`STM32F0 Cube package advise to use 'stm32f030x6' code for both stm32f030x4 and stm32f030x6 soc variants.`